### DISCUSSION TITLE
Welcome/Bienvenue/Bienvenid@ to the Chamilo Discusssions space!

### DISCUSSION THREAD

**Discussion by ywarnier** [chamilo/chamilo-lms#5139] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and as a replacement for forum.chamilo.org. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  
  This is a multilingual forum, so don't be offended if you see messages in another language (use a translator if needed) and don't hesitate to post in your own language if you feel it's easier for you to express yourself that way. Just don't mix languages in the same message.

  To get started, comment below with an introduction of yourself and tell us about the skills you have that are relevant to this community.
  
  This space is a community exchange space. Thank you for keeping it ad-free. Commercial messages, direct or indirect, will be removed on detection (no warning). If the user doesn't understand after this first strike, his/her account will be blocked from this forum.
  
  ## 👋 Bienvenid@!
  Estamos usando Discussions como un lugar para conectar con otros miembros de nuestra comunidad y como un remplazo para forum.chamilo.org. Esperamos que puedas:
  * Preguntar lo que te genera dudas.
  * Compartir ideas.
  * Involucrarte con otros miembros de la comunidad.
  * Dar una calurosa bienvenida a los demás. Recuerda que construimos esta comunidad juntos 💪.

  Este es un foro multilingue, así que no te sientas ofendid@ si ves mensajes en otro idioma (usa un traductor si necesitas) y no dudes en publicar mensajes en tu propio idioma si piensas que es más fácil para ti expresarte así. Solo evita mezclar los idiomas en un mismo mensaje (lo hace más difícil traducir).

  Para iniciar, puedes comentar aquí con una breve introducción personal que incluya una pequeña descripción de tus habilidades relevantes a esta comunidad.
  
  Este espacio es un espacio de intercambio comunitario. Gracias por manter tu publicidad de servicios profesionales fuera de este foro. Los mensajes comerciales directos o indirectos serán eliminados la primera vez, sin advertencia. Si la persona no entiende, su cuenta será excluida de este foro.
  
  ## 👋 Bienvenue !
Nous utilisons Discussions comme un endroit pour se connecter avec les autres membres de notre communauté et comme un remplacement pour forum.chamilo.org. Nous espérons que tu pourras :
* Poser des questions sur ce qui tr pose problème.
* Partager des idées.
* T'impliquer avec les autres membres de la communauté.
* Accueillir chaleureusement les autres. N'oublies pas que nous construisons cette communauté ensemble 💪.

C'est un forum multilingue, donc ne te sens pas offensé si tu vois des messages dans une autre langue (utilise un traducteur si nécessaire) et n'hésites pas à publier des messages dans ta propre langue si c'est plus facile de t'exprimer comme ça. Évite simplement de mélanger les langues dans un même message (cela rend la traduction plus difficile).

Pour commencer, tu peux commenter ici avec une brève introduction personnelle incluant une petite description de tes compétences liées à cette communauté.

Cet espace est un espace d'échange communautaire. Merci de garder la publicité de tes services professionnels hors de ce forum. Les messages commerciaux directs ou indirects seront supprimés la première fois, sans avertissement. Ensuite, le compte de la personne n'ayant pas compris sera exclus du forum.

**Comment by ywarnier** [comment]
Hi, I'm Yannick, I lead the Chamilo project since its birth in 2010. Welcome!

Feel free to ask questions about whatever you want to know in Chamilo, in the "Q&A" category. Try to be descriptive (provide a lot of details) so everyone can understand your question fully and give you the best possible answer.

**Reply by abelgzalazar** [reply]
Hola queria saber si con chamilo , puedo ingresar al campus de mi universidad https://ulp.untrefvirtual.edu.ar/
Desde ya muchas gracias.

**Reply by Kaneda-1** [reply]
Hola @abelgzalazar la dirección que indicas no es un aula Chamilo.
Contacta con tu Universidad.
Saludos.

**Reply by T3dpikul** [reply]
Hola, estoy trabajando con la versión de Chamilo 1.11.26, por temas de trabajo me están solicitando una opción en que los archivos que se suben en la carpeta "documento" que tiene cada curso, no puedan ser descargados, que solo puedan ser revisados de manera online dentro de Chamilo, he revisado en toda la configuración dentro de Chamilo y no encontré ninguna opción donde deshabilitar el botón de guardar

Lo que sí encontré es que se pueden marcar con una marca de agua los archivos en PDF, la cual también configure y no funciona, si me ayudaran sería de gran ayuda

Saludos

Hello, I am working with Chamilo version 1.11.26. For work-related reasons, I have been asked for an option where the files uploaded in the "document" folder of each course cannot be downloaded, and can only be viewed online within Chamilo. I have reviewed the entire configuration within Chamilo and did not find any option to disable the save button.

What I did find is that PDFs can be watermarked, which I also configured, but it doesn't work. If you could help me, it would be of great assistance.

Best regards,

**Reply by rosalmuj75-alt** [reply]
Saludos amigos, desde el día 28 de agosto, no he podido acceder a las actividades de las secciones de curso de formación de la enseñanza de la lengua, logré responder las auto evaluaciones, pero he presentado problemas para resolver los ejercicios de analisis porque me salen las preguntas, pero no me da para escribir. Que hago.
Lo otro es las aulas virtuales para el debate colectivo por donde ingreso para estás aulas virtuales, estoy bastante desorientada.

**Comment by IoTSIGithub** [comment]
Hello Where have all the previous posts gone?

**Comment by jgnosolored** [comment]
Old forum posts are in https://forum.chamilo.org/

**Comment by DJIEH** [comment]
Bonjour , je suis ingénieur des technologies éducatives , j'aimerai savoir si champion organise des formations de formateurs ou a des certificats pour des formateurs chamilio

**Comment by Oscarlpcv** [comment]
I need to implement a 2 month course focused on Environmental Education for 5000 teachers of public secondary schools in Mexico, (+ the project staff, approximately 20 people) Is Chamilo suitable? If yes, which Moodle functionalities would be enough for an optimal implementation of the course and management of the implementation team? If yes, what plan do you recommend me to acquire? What are the total costs?

**Reply by Raymond-Onwude** [reply]
Yes, Chamilo is an excellent choice for your two-month environmental education course targeting 5,000 teachers and 20 project staff. Its scalable, open-source platform comes with essential e-learning features and robust tools for effective course management.

### **Key Features for Optimal Course Implementation**
- **Learning Paths:** Create structured course modules for easy navigation.
- **Assignments & Quizzes:** Comprehensive tools for assessments and progress tracking.  
- **BigBlueButton (BBB) Integration:** Free and ideal for virtual sessions, offering video conferencing for interactive classes.
- **Certificate Module:** Automatically generate custom certificates for course completion.
- **Gamification Plugin:** Boost engagement through badges and leaderboards.  
- **SCORM & xAPI Support:** Deliver dynamic, interactive learning content.  
- **Surveys & Polls:** Collect feedback to optimize course delivery.  
- **Document Repository:** Efficiently manage and share course materials.  
- **Social Media Integration:** Encourage course completion recognition via social platforms.  
- **E-Commerce Integration:** Sell additional courses directly if needed.

Best Regards, 
Raymond Onwude

**Comment by ywarnier** [comment]
Hi. I have updated my initial message to add a note about commercial advertisement of products or services. This is a community space. If you are not able to understand and respect this, your messages are not welcome. Please promote your services on your own website and provide quality content there so people actually find it. Abusing this forum will not be tolerated.

**Reply by Raymond-Onwude** [reply]
Thanks for your feedback. I understand the importance of keeping this forum focused on meaningful discussion and knowledge sharing without commercial promotion. I apologize if my response seemed out of place, as my goal was purely to help the person asking the question based on my experience with Chamilo.

Moving forward, I’ll make sure my contributions are focused solely on providing value to the community, without any promotional intent. I truly believe in the potential of Chamilo and am passionate about supporting its growth. I’ll be more mindful of the community guidelines and ensure that any reference to my services is reserved for more appropriate channels.

Thanks for your understanding, and I look forward to continuing to be a positive part of this community.

**Comment by Gerald151601** [comment]
Buenas tardes, me registre en el sistema hace un mes aproximadamente pero nunca me llegó correo! Intento ingresar y me dice que no me encuentro en la lista de elegibles! Alguien me podría orientar 🙏🏼 de antemano muchas gracias

**Reply by Kaneda-1** [reply]
A ver.

"Me registré", "no llegó correo", "lista de elegibles"... ¿En qué sistema? ¿Qué lista? No somos adivinos.

Si quieres ayuda, danos datos. ¿Dónde te apuntaste? ¿La URL exacta? ¿Qué es esa "lista de elegibles" que te excluye?

Sé preciso. Explica bien el problema.

**Comment by federico-digito** [comment]
Buongiorno a tutti  ho installato Chamilo per test su server Ubuntu VM 25.0 , ma non riesco a configurare il sistema  per inserire file piu grandi di 2 MB nei corsi Potete aiutarmi ?
Saluti

**Comment by tovarkarla711-cloud** [comment]
Buenos dias, quisiera saber cómo hago yo que un día estaba haciendo el curso, y la página se puso lente y aparte se me fue la luz en mi casa y no pude terminar de responder la evaluación, será que me daran nuevamente otra oportunidad? O como hago por qué salí aplazada sin querer

**Comment by escolvig** [comment]
Buenas tardes, estoy intentando subir unos documentos en pdf para un curso, pero me aparece error, tengo la plataforma actualizada y aun asi no me permite subir los archivos, que debo hacer.
Gracias !

**Comment by peninsula3** [comment]
For Chamilo 1.11.26, there is no built-in option to prevent downloads of course documents. Files in the “document” folder can only be watermarked but full download prevention is not supported. The best option is to use a PDF viewer plugin or host files in a secure external viewer that only allows online viewing.

**Comment by isaacsanchez0361-byte** [comment]
hola, que ha pasado con los demás cursos?? quiero seguir en los estudios. como continuo???

**Comment by danieldocumentos** [comment]
Me gustaría que agreguen la opción de categorías en Lecciones

**Comment by LeonardoCarone** [comment]
Buenas tardes, quisiera saber que recomiendan con respecto a la version 2.0.3, la he estado probando para ver si la implemento pero me he llevado varias sorpresas encontrando numerosos bugs, algunos ya resueltos en commits a la rama master.

Por lo que quisiera saber si una versión estable de la 3.0 está proxima, para intentar organizarme a ver si sigo corrigiendo esta versión que tengo en el laboratorio o me lanzo a probar la beta de la 3.0.

Me serviría muchísimo una recomendación, muchas gracias.
